errors.js

```diff
@@ -0,0 +1,12 @@
+exports.handleCustomError = (req, res, err) => {
```

Comment:
We currently don't have any custom errors being implemented so we don't need this function yet